### ISSUE TITLE
Parent vs Salt data

### ISSUE THREAD

**Issue by agaulton** [chembl/GLaDOS#322]
In current interface, we only return parent compounds from searches etc, and aggregate all of the data from all the salts. You can see the salt only if you go to the report card and choose alternative forms/go to the page directly with the salt ChEMBL ID, or as a column on the bioactivity summary.

In the new interface, we are returning all forms of the compound in the search results (e.g. try 'quinine'), rather than just the parent. But also we are not aggregating the data (e.g., on the sildenafil report card you see only data for sildenafil itself and not sildenafil citrate, which means the mechanism of action is not shown for example).

While it's simpler to keep things separate, I think mostly users want to consider all of the salt forms when they look at a parent compound (e.g., not miss the 41 activities that were measured against the citrate when they look at sildenafil)...

**Comment by mnowotka** [comment]
Isn't it a duplicate of #300?

**Comment by agaulton** [comment]
Yes, but on current interface search results are merged as well as data on report card.
